spack/packages/dla-future/package.py

```diff
@@ -84,12 +85,11 @@ def cmake_args(self):
         # CUDA/HIP
         args.append(self.define_from_variant("DLAF_WITH_CUDA", "cuda"))
         args.append(self.define_from_variant("DLAF_WITH_HIP", "rocm"))
-
-        # HIP support requires compiling with hipcc
-        if '+rocm' in self.spec:
-            args += [self.define('CMAKE_CXX_COMPILER', self.spec['hip'].hipcc)]
-            if self.spec.satisfies('^cmake@3.21.0:3.21.2'):
-                args += [self.define('__skip_rocmclang', True)]
+        if "+rocm" in spec:
+            archs = self.spec.variants["amdgpu_target"].value
+            if archs != "none":
+                arch_str = ";".join(archs)
+                args.append(self.define("CMAKE_HIP_ARCHITECTURES={0}".format(arch_str)))
```

Comment:
I think this is what is causing the current configuration error in CI:
```suggestion
                args.append("-DCMAKE_HIP_ARCHITECTURES={0}".format(arch_str))
```